common.css

```diff
@@ -0,0 +1,59 @@
+:root {
+    --clr-grey-background: #f3f3f3;
+}
+
+.ecom-nav {
+    min-height: 7rem;
+    background-color: var(--white);
+    z-index: 99999;
```

Comment:
Avoid using large Z-index value.

Reply:
Yes, I wanted the navbar to be on top always, hence the large value. Will change it if a need arises.

Reply:
Ok